iceoryx_hoofs/include/iceoryx_hoofs/cxx/newtype.hpp

```diff
@@ -54,6 +54,14 @@ namespace cxx
 ///         //                 otherwise the code will not compile
 ///             using ThisType::ThisType;   // this makes all constructors of NewType available for Index
 ///             using ThisType::operator=;  // put the assignment operators in scope
+///
+///             // implement ctors and assignment operators when they are implemented by the base class
+///             // this is necessary pro prevent warnings from specific compilers
```

Comment:
Do you mean `to prevent warnings`.

What kind of mistake is this when one writes a complete different word ... I say you made a moose.

Reply:
How about `from some compilers`?